src/GPS/GPSProvider.cc

```diff
@@ -72,6 +85,13 @@ void GPSProvider::run()
         }
         gpsDriver->setSurveyInSpecs(_surveyInAccMeters * 10000, _surveryInDurationSecs);
 
+        // fixed base position:
+        double latitude = xyz;
+        double longitude = xyz;
+        float altitude = xyz;
+        float position_accuracy = xyz;
+        gpsDriver->setBasePosition(latitude, longitude, altitude, position_accuracy);
```

Comment:
@bkueng What is the position_accuracy field here?

Reply:
You get that from the survey-in status: https://github.com/PX4/GpsDrivers/blob/master/src/gps_helper.h#L112. Not sure how important it is. If unknown, I'd use something like 50cm.

Reply:
Wondering if really needed? Poking around in Mission Planner UI which already support this feature shows that it doesn't have this value.

Reply:
You need to provide a value for the u-blox when setting the position. So I would say we should do it properly and use the correct value (since we already have it).

Reply:
Got it

Reply:
for setBasePosition call is position_accuracy in mm?

Reply:
Yes in mm: https://github.com/PX4/GpsDrivers/blob/master/src/base_station.h#L76